app/controllers/assets.php

```diff
@@ -0,0 +1,34 @@
+<?php
+class assets extends Controller {
+    function __construct() {
+        $this->load_model('assets_model');
+    }
+
+    function index() {
+        $this->load_view('assets/index');
+    }
+
+    function insert() {
+        $isSuccess = null;
```

Comment:
@imjalpreet Just a suggestion, but since the framework is written in underscore naming convention, i think its better to continue that and not use camelCase. This is mostly for consistency. But its fine if u don't want to change it. As i said, its just a suggestion.